### DISCUSSION TITLE
Date range from zoomed plot

### DISCUSSION THREAD

**Discussion by HakanL** [datalust/seq-tickets#1577] [Help]
I have a task where I need to find long-running requests. I can plot the duration over a week and can easily see the peaks. I then zoom in to find the time range where it happens. But how can I transfer the time range displayed into the query? I basically want to go from a group by time(30m) to the list of events in the view of the graph. Of course I could type it in, but it would be great to have a method to quickly get to the source data (or time range) from the plot view.

![image](https://user-images.githubusercontent.com/407941/170874843-d193f621-3e1e-45b3-ba62-46f2b2f1d77b.png)

![image](https://user-images.githubusercontent.com/407941/170874914-9876d344-46c0-441c-ad78-c28da64f7b3b.png)

**Comment by nblumhardt** [comment]
Agreed that this would be great 👍 - currently there's no way to synchronize these things, unfortunately.
